src/modules/songs/forms/AddSongToSet/components/SetSelectionUpcomingSets.tsx

```
@@ -0,0 +1,87 @@
+  } = api.set.getUpcoming.useQuery({
+    organizationId: userMembership?.organizationId,
+  });
+
+  if (isUpcomingSetsQueryLoading) {
+    return <Skeleton />;
+  }
+
+  if (!!upcomingSetsQueryError || !upcomingSetsData) {
+    // TODO: how do we better handle this?
+    toast.error(
+      `Ran into an issue when trying to find the upcoming sets${!!upcomingSetsQueryError ? `: ${upcomingSetsQueryError.message}` : ""}`,
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)